packages/dashboard/jest.config.ts

```diff
@@ -28,12 +28,14 @@ export default {
       },
     ],
   },
-  transformIgnorePatterns: [],
   setupFilesAfterEnv: ['mutationobserver-shim'],
   //transform: {
   //   '.+\\.ts$': 'ts-jest',
   //   '^.+\\.tsx?$': 'ts-jest',
   //   '^.+\\.(js|jsx)$': 'babel-jest',
   // },
-  // transformIgnorePatterns: ['node_modules/(?!@awsui/components-react)/'],
+  transformIgnorePatterns: [
```

Comment:
this issue has been fixed by introducing `@swc/jest` at this [commit](https://github.com/awslabs/iot-app-kit/commit/1a4f2a490ada12135f9591b2c6b0716bcc31a004) 
you might want to just rebase on that

Reply:
👍 perfect, I had introduced this to fix an issue when I added popperjs to the package. Looks like that makes this ignore unnecessary.